pkg/server/data/client/client_http.go

```diff
@@ -0,0 +1,114 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/nrc-no/core/pkg/server/data/api"
+)
+
+type httpClient struct {
+	baseURL string
+	client  *client
+}
+
+const tablesUrl = "/apis/data.nrc.no/v1/tables"
+const mimeJson = "application/json"
+const headerAccept = "Accept"
+const headerContentType = "Content-Type"
+
+type HTTPClient interface {
+	CreateTable(ctx context.Context, request api.Table) (api.Table, error)
+	GetRecord(ctx context.Context, request api.GetRecordRequest) (api.Record, error)
+	GetRecords(ctx context.Context, request api.GetRecordsRequest) (api.RecordList, error)
+	GetTables(ctx context.Context, request api.GetTablesRequest) (api.GetTablesResponse, error)
+	GetTable(ctx context.Context, request api.GetTableRequest) (api.Table, error)
+	GetChanges(ctx context.Context, request api.GetChangesRequest) (api.Changes, error)
+	PutRecord(ctx context.Context, request api.PutRecordRequest) (api.Record, error)
+}
```

Comment:
it seems like there's no method to create a new record? am I missing something, or is there a trick?

Reply:
`PutRecord` ;)

Reply:
ah, the id in the request irritated me